docs/collections/_schema/human-readable-schema-grammar.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Human-Readable schema grammar
+nav_order: 2
+---
+<!-- markdownlint-disable-file MD040 -->
+
+# Grammar specification for the human-readable schema format {#schema-grammar}
+{: .no_toc }
+
+This topic describes the grammar specification for the human-readable schema format. For a more complete description, see [Schema format](../schema/human-readable-schema-format).
+
+The grammar applies the following the conventions. Capitalized words stand for grammar productions, and lexical tokens are given in all-caps. When productions or tokens match those in the Cedar policy grammar, we use the same names (e.g., `IDENT` and `Path`).
```

Comment:
"the same names" - The same as what?

Reply:
"grammar productions" and "lexical tokens" aren't defined here or on the schema format page. What do they mean?

Reply:
The names on the left hand side of grammars. I'd prefer to keep "grammar productions" as it's a common technical term.